docs/tuning-apps/large-clusters/libraries.rst

```diff
@@ -0,0 +1,52 @@
+Libraries
+=========
+
+Location: network vs. local filesystems
+---------------------------------------
+
+Open MPI itself doesn't really care where its libraries and plugins
+are stored.  However, where they are stored does have an impact on
+startup times, particularly for large clusters, which can be mitigated
+somewhat through use of Open MPI's configuration options.
+
+Startup times will always be minimized by storing the libraries and
+plugins local to each node, either on local disk or in ramdisk. The
+latter is sometimes problematic since the libraries do consume some
+space, thus potentially reducing memory that would have been available
+for MPI processes.
+
+There are two main considerations for large clusters that need to
+place the Open MPI libraries on networked file systems:
+
+* While dynamic shared objects ("DSO") are more flexible, you
+  definitely do *not* want to use them when the Open MPI libraries
+  will be mounted on a network file system! Doing so will lead to
+  significant network traffic and delayed start times, especially on
+  clusters with a large number of nodes. Instead, be sure to configure
+  your build with ``--disable-dlopen``. This will include the DSO's in
```

Comment:
```suggestion
  your build with :ref:`--disable-dlopen<label-building-installation-cli-options>`. This will include the DSO's in
```

It might be worth adding a new label in installing-open-mpi/configure-cli-options/installation.rst to directly link to the disable dlopen section

Reply:
Good idea, done (in my next commit)